src/Symfony/Contract/README.md

```diff
@@ -0,0 +1,69 @@
+Symfony Contracts
+=================
+
+A set of abstractions extracted out of the Symfony components.
+
+Can be used to build on semantics that the Symfony components proved useful - and
+that already have neat implementations if you don't mind.
+
+Design Principles
+-----------------
+
+ - contracts are split by domain, each into their own sub-namespace;
+ - contracts are self-consistent sets of PHP interfaces, traits, normative
+   docblocks and reference test suite when applicable, etc.;
+ - all contracts must have a proven implementation to enter this repository;
+ - they must be backward compatible with existing Symfony components.
+
+FAQ
+---
+
+### How to use this package?
+
+The abstractions in this packages are useful to achieve loose coupling and
```

Comment:
package?

Reply:
Package kind of makes sense here IMO, we distribute it on its own as a component?